index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Panda_login</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+  <div class="container">
+    <div class="logo"><a href="/">
+    <img src="images/logo/logo.svg" width="396px" height="132px"></a>
```

Comment:
💊 제안
현재 코드에서는 두 태그가 구분없이 한 줄로 붙어 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~
```suggestion
    <div class="logo">
      <a href="/">
        <img src="images/logo/logo.svg" width="396px" height="132px">
      </a>
```